assets/chef-recipes/cookbooks/connector_odbc/recipes/install_dependencies.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+package 'tar'
+if %w[debian ubuntu].include?(node[:platform])
```

Comment:
Use [platform?](https://docs.chef.io/infra_language/checking_platforms/) for checking the target platform in chef resources.